src/pages/home/HeaderView.js

```diff
@@ -50,69 +51,124 @@ const defaultProps = {
                             >
-                                <Icon src={Pin} fill={props.report.isPinned ? themeColors.heading : themeColors.icon} />
+                                <MultipleAvatars avatarImageURLs={this.props.report.icons} />
+                                <View style={[styles.flex1, styles.flexRow]}>
+                                    <OptionRowTitle
+                                        option={this.reportOption}
+                                        tooltipEnabled
+                                        numberOfLines={1}
+                                        style={[styles.headerText]}
+                                    />
+                                </View>
                             </Pressable>
+                            <View style={[styles.reportOptions, styles.flexRow, styles.alignItemsCenter]}>
+                                {this.props.report.hasOutstandingIOU && (
+                                    <IOUBadge iouReportID={this.props.report.iouReportID} />
+                                )}
+                                <View
+                                    ref={el => this.videoChatIconWrapper = el}
+                                    onLayout={this.measureVideoChatIconPosition}
+                                >
+                                    <Pressable
+                                        onPress={() => {
+                                            this.toggleVideoChatMenu();
+                                        }}
+                                        style={[styles.touchableButtonImage, styles.mr0]}
+                                    >
+                                        <Icon
+                                            src={Phone}
+                                            fill={this.state.isVideoChatMenuActive
+                                                ? themeColors.heading
+                                                : themeColors.icon}
+                                        />
+                                    </Pressable>
+                                </View>
+                                <Pressable
+                                    onPress={() => togglePinnedState(this.props.report)}
+                                    style={[styles.touchableButtonImage, styles.mr0]}
+                                >
+                                    <Icon
+                                        src={Pin}
+                                        fill={this.props.report.isPinned ? themeColors.heading : themeColors.icon}
+                                    />
+                                </Pressable>
+                                <VideoChatMenu
+                                    onClose={this.toggleVideoChatMenu}
+                                    isVisible={this.state.isVideoChatMenuActive}
+                                    anchorPosition={this.state.videoChatIconPosition}
+                                />
```

Comment:
After looking at these changes... I'm curious whether it makes more sense to just make `VideoChatMenu` and the button that launch it the same component so that things are a bit more self contained? Nothing else besides these two components needs to know `isVideoChatMenuActive`.